10-zwave.js

```diff
@@ -410,7 +410,16 @@ module.exports = function(RED) {
 					) {
 						if (debug) console.log('attempting direct call to OpenZWave API: %s(%s)', msg.topic, payload);
 						try {
-							ozwDriver[msg.topic].apply(ozwDriver, payload);
+							var retval = ozwDriver[msg.topic].apply(ozwDriver, payload);
+							var newmsg = {'topic': msg.topic};
```

Comment:
What about *async* calls to OpenZWave? You won't be getting a return vallue from the call here, but through a callback some milliseconds later (thus: from the zwave-in node)